test/client/scraper/test_youtube_metadata.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+import json
+from mock import patch
+from client.scraper.youtube_metadata import YoutubeMetadata
+from client.response import Response
+from client.constants import UrlTypes
+
+
+class TestYoutubeMetadata(unittest.TestCase):
+
+    @patch.object(YoutubeMetadata, 'get_json_response')
+    def test_drive(self, response_json):
+        with open("test/mocks/youtube.json", "r") as testFile:
+            data = testFile.read().replace('\n', '')
+
+        response_json.return_value = json.loads(data)
+        scraper = YoutubeMetadata()
+
+        url_data = Response()
+        url_data.set_content("", data, "", "", UrlTypes.YOUTUBE)
+
+        response = json.loads(scraper.parse_content(url_data))
+        self.assertEqual(response.get("title"), "Cutest Baby Talk Ever!")
```

Comment:
I think it doesn't really matter that the title is an expected value, since we are not manipulating the title in anyway. Instead you should check that specific values exist and that that we recognize the type of the video (youtube).

Instead do something like 
<code>
 self.assertTrue(response.get("title")) // unsure if assertTrue works. Basically it should assert that the response is not null or empty.
self.assertTrue(response.get("description")) 
self.assertTrue(response.get("title")) 
</code>